src/test/java/com/iexec/worker/chain/WebSocketBlockchainListenerTests.java

```diff
@@ -57,9 +57,11 @@ class WebSocketBlockchainListenerTests {
 
     @DynamicPropertySource
     static void registerProperties(DynamicPropertyRegistry registry) {
-        registry.add("core.protocol", () -> "http");
-        registry.add("core.host", () -> environment.getServiceHost(CORE_SVC_NAME, CORE_SVC_PORT));
-        registry.add("core.port", () -> environment.getServicePort(CORE_SVC_NAME, CORE_SVC_PORT));
+        final String coreHost = environment.getServiceHost(CORE_SVC_NAME, CORE_SVC_PORT);
+        final int corePort = environment.getServicePort(CORE_SVC_NAME, CORE_SVC_PORT);
+        final String coreUrl = "http://" + coreHost + ":" + corePort;
+
+        registry.add("core.url", () -> coreUrl);
```

Comment:
Use `getServiceUrl` helper here (see blockchain node address override below)
```suggestion
        registry.add("core.url", () -> getServiceUrl(coreHost, corePort));
```